test/integration/ads_integration_test.cc

```diff
@@ -645,7 +645,7 @@ TEST_P(AdsIntegrationTest, XdsBatching) {
     static_resources->add_listeners()->MergeFrom(buildListener("rds_listener2", "route_config2"));
   });
 
-  pre_worker_start_test_steps_ = [this]() {
+  server_init_coroutines_ = [this]() {
```

Comment:
Not a fan of coroutine, since that makes me think of Go. Can we call it `on_server_init_fr_`, unless you do intend to invoke some kind of concurrency idea due to the coordinating role these routines play?

Reply:
+1 to Alyssa's comment--I'm happy to change the name, but I want to understand the meaning of the name I'm changing it to :-}.

Reply:
Sorry, should have been `on_server_init_fn_`!

Reply:
My days of character-limited FORTRAN being long gone, I'm still going to argue for function_  :-P

Reply:
Sure, makes sense.

Reply:
Done.